.rspec

```diff
@@ -1,4 +1,3 @@
 --color
 --format documentation
---order default
```

Comment:
This option was deprecated and caused `rspec` to fail. The default option is now called `defined` ([see docs](https://rspec.help/rspec/command-line/)), so we don't need to specify it.

Reply:
It looks like [this happened in Rspec v3](http://rspec.info/blog/2014/05/notable-changes-in-rspec-3/), which is _quite_ old. I'm surprised we didn't notice warnings or issues from the beginning 😅